userappwithbroker/src/main/java/com/microsoft/aad/adal/example/userappwithbroker/Constants.java

```diff
@@ -47,11 +42,8 @@ public String getText() {
 
     enum DataProfile {
         GRAPH("https://graph.windows.net"),
-        SHAREPOINT_MS_DEV("https://msdevex-my.sharepoint.com"),
-        SHAREPOINT("00000003-0000-0ff1-ce00-000000000000"),
-        OFFICE_ONEDRIVE("6a9b9266-8161-4a7b-913a-a9eda19da220"),
-        SIMPLE("00000002-0000-0000-c000-000000000000");
```

Comment:
Why are these removed?

Reply:
I know for sure that `SHAREPOINT` and `SIMPLE` are used in a few different tests. I'm not inclined to removing these as we may end up needing them in the future.